src/System.Collections.Specialized/tests/NameValueCollection/CtorIntIKCTests.cs

```diff
@@ -152,7 +149,7 @@ public void Test01()
                 {
                     Assert.False(true, string.Format("Error, returned null instead of {0} ", exp3));
                 }
-                else if (String.Compare(nvc["Everything Is Equal!"], exp3) != 0)
+                else if (string.Compare(nvc["Everything Is Equal!"], exp3) != 0)
```

Comment:
The vast majority of the changes in this PR are search-and-replace / refactorings to change type names into keywords, e.g. ```String``` to ```string```.  In the future, if you're going to make such changes (we've actually shied away from having such changes done as we plan to do a full sweep for such things programmatically at some point), please do them in a separate commit, as doing so in a separate commit will make it easier to review both these programmatic changes and the few others that aren't.  Thanks.